packages/render/src/node/read-stream.ts

```diff
@@ -11,8 +11,8 @@ export const readStream = async (
 ) => {
   let result = '';
 
-  if ('pipeTo' in stream) {
-    // means it's a readable stream
+  if ('pipeTo' in stream && typeof WritableStream !== 'undefined') {
```

Comment:
no need to check if WritableStream is available again here

```suggestion
  if ('pipeTo' in stream) {
```